CMakeLists.txt

```diff
@@ -490,6 +490,10 @@ list(APPEND draco_unity_plug_sources
             "${draco_src_root}/unity/draco_unity_plugin.cc"
             "${draco_src_root}/unity/draco_unity_plugin.h")
 
+list(APPEND draco_unity_enc_plug_sources
```

Comment:
Please run cmake-format on the CMake files. You can install 0.5.1 using pip via 'pip install cmake-format==0.5.1'.

Reply:
Will do!
Note: cmake-format did changes in sections not touched by me. I'll only commit changes to lines I added/changed.